OpenDreamRuntime/Procs/DMOpcodeHandlers.cs

```diff
@@ -1143,8 +1143,7 @@ public static ProcStatus Divide(DMProcState state) {
                     state.Push(new DreamValue(0));
                     break;
                 case DreamValue.DreamValueType.Float when second.IsNull:
-                    state.Push(new DreamValue(first.MustGetValueAsFloat()));
-                    break;
+                    throw new Exception("Division by null");
```

Comment:
Can't hurt to have information about what we're trying to divide
```suggestion
                    throw new Exception($"Attempted to divide {first} by null");
```